pkgs/by-name/ne/nerdlog/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+buildGoModule (finalAttrs: {
+  pname = "nerdlog";
+  version = "1.7.2";
+
+  src = fetchFromGitHub {
+    owner = "dimonomid";
+    repo = "${finalAttrs.pname}";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-5duKO5Ep++M+I8TXhtFpWydikzMC1AQefzFQEQjbeq4=";
+  };
+
+  env.CGO_ENABLED = 0;
+  subPackages = [ "cmd/nerdlog" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/dimonomid/nerdlog/version.version=${finalAttrs.version}"
+    "-X github.com/dimonomid/nerdlog/version.commit=${finalAttrs.src.rev}"
+    "-X github.com/dimonomid/nerdlog/version.date=1970-01-01T00:00:00Z"
+    "-X github.com/dimonomid/nerdlog/version.builtBy=nix@nixpkgs"
+  ];
+
+  vendorHash = "sha256-J3u8MXUXoMttj4dVKAcuMvqCOsbYzHnulunXxprV8hA=";
+
+  passthru.updateScript = nix-update-script { };
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+    HOME=1
+    $out/bin/nerdlog --version | grep "${finalAttrs.version}"
+    runHook postInstallCheck
+  '';
```

```suggestion
  doInstallCheck = true;
  nativeInstallCheckInputs = [
    writableTmpDirAsHomeHook
    versionCheckHook
  ];
  versionCheckDontIgnoreEnvironment = true
```

I haven't tested, but this should be enough.

---

yes, followed the rust examples, found the reason:
versionCheckDontIgnoreEnvironment (https://github.com/NixOS/nixpkgs/pull/403971) 
does not even made it into master, but is still stuck in staging limbo

---

oh crap :(

---

this problem will solve itself, just by sitting here and wait, that's the best kind of problems :)  
[latest deadline will be: 23.05.2025]

---

If you don't mind waiting for it, it's fine for me.